roles/security_errata/tasks/main.yml

```diff
@@ -1,11 +1,13 @@
 ---
-- name: Patch CVE-2014-0160
+- name: update apt index
+  apt: update_cache=yes cache_valid_time=3600
+
+- name: update SSL
   apt: pkg={{ item }} state=latest
   with_items:
   - libssl1.0.0
   - openssl
```

Comment:
Who decides what latest is though? I am unfamiliar with how apt works, if one gets mirrored to an out of date mirror, would it see the old openssl as "latest" and think all is good?